apps/services/sessions/infra/sessions.ts

```diff
@@ -1,8 +1,17 @@
-import { json, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+import { PersistentVolumeClaim } from '../../../../infra/src/dsl/types/input-types'
+import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
```

Comment:
Is the ref import needed? Looks possibly unused

Reply:
well spotted, I was attempting to use `ref()` instead of passing the service as a param.